tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -1004,10 +1004,13 @@ private void checkLocaleOk(final String locale, final boolean expectedInCLDRLoca
      * Needs to be changed for each Limited Submission cycle. Is ignored for full submission cycles.
      */
     final Set<String> SAMPLE_EXCEPTIONAL_PATHS = ImmutableSet.of(
-        "//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"length-kilometer\"]/gender"
+        "//ldml/personNames/nameOrderLocales[@order=\"givenFirst\"]",
+        "//ldml/personNames/nameOrderLocales[@order=\"surnameFirst\"]",
+        "//ldml/localeDisplayNames/territories/territory[@type=\"TR\"]",
+        "//ldml/localeDisplayNames/territories/territory[@type=\"TR\"][@alt=\"variant\"]"
         );
 
-    final String sampleDisallowedInLimitedSubmission = "//ldml/annotations/annotation[@cp=\"🎅\"]";
+    final String sampleDisallowedInLimitedSubmission = "//ldml/localeDisplayNames/territories/territory[@type=\"SY\"]";
```

Comment:
Not a blocker, but we should probably spruce this up with a more extensive test. I'm thinking a list like:
 {path, allowedLocales, disallowedLocales},
...

Like
 {"//ldml/personNames/nameOrderLocales[@order=\"givenFirst\"]", "en fr", "lmo"},
{"//ldml/localeDisplayNames/territories/territory[@type=\"SY\"]", "", "en fr lmo"},
...

Reply:
I want to move this to junit and use a CSV file ! Agreed